pkg/scheduler/core/generic_scheduler.go

```diff
@@ -677,13 +677,13 @@ func EqualPriorityMap(_ *v1.Pod, _ interface{}, nodeInfo *schedulercache.NodeInf
 // 3. Ties are broken by sum of priorities of all victims.
 // 4. If there are still ties, node with the minimum number of victims is picked.
 // 5. If there are still ties, the first such node is picked (sort of randomly).
```

Comment:
Please add a comment and mention that "minNodes1" and "minNodes2" are being reused in this function to save on memory allocation and garbage collection time.

Reply:
Added, thanks for your review